src/components/ButtonPannel.js

```diff
@@ -0,0 +1,37 @@
+import Button from './Button';
+
+export default function ButtonPannel() {
+  return (
+    <div className="ButtomPannelContainer">
+      <div className="rowOne">
+        <Button name="AC" />
+        <Button name="+/-" />
+        <Button name="%" />
+        <Button name="÷" />
+      </div>
+      <div className="rowTwo">
+        <Button name="7" />
+        <Button name="8" />
+        <Button name="9" />
+        <Button name="X" />
+      </div>
+      <div className="rowThree">
+        <Button name="4" />
+        <Button name="5" />
+        <Button name="6" />
+        <Button name="-" />
+      </div>
+      <div className="rowFour">
+        <Button name="1" />
+        <Button name="2" />
+        <Button name="3" />
+        <Button name="+" />
+      </div>
+      <div className="rowFive">
+        <Button name="0" />
+        <Button name="." />
+        <Button name="=" />
+      </div>
```

Comment:
- [ ] Rather than doing this consider grouping the buttons then mapping through the group to display them it's neater:
```
Group 1: AC, +/-, %, ÷
Group 2: 7, 8, 9, X
Group 3: 4, 5, 6, -
Group 4: 1, 2, 3, +
Group 5: 0, ., =
```